### PR TITLE
chore(pnpm): migrate to pnpm v7 🎉

### PR DESCRIPTION
**Pull request by BlackHole1**
Ref: 
  * https://github.com/netless-io/flat/pull/1510
  * https://github.com/pnpm/pnpm/releases/tag/v7.0.0
  * https://www.raulmelo.dev/blog/migrating-from-pnpm-6-to-7

### CODE REVIEW

**Review by hyrious — DISMISSED** [review]

File: .github/workflows/check-code.yml [hunk 1]
```diff
@@ -66,15 +66,14 @@ jobs:
             ./packages/**/node_modules
 
       - name: Install PNPM
-        uses: pnpm/action-setup@v2.0.1
+        uses: pnpm/action-setup@v2.2.1
```
Comment (LitoMore) [inline .github/workflows/check-code.yml]:
```suggestion
        uses: pnpm/action-setup@v2
```

File: .github/workflows/check-code.yml [hunk 2]
```diff
@@ -66,15 +66,14 @@ jobs:
...
         with:
-          version: latest
-          run_install: false
+          version: 7.0.0
```
Comment (LitoMore) [inline .github/workflows/check-code.yml]:
```suggestion
          version: 7
```

File: .github/workflows/deploy-storybook.yml [hunk 3]
```diff
@@ -26,26 +26,25 @@ jobs:
             ./packages/**/node_modules
 
       - name: Install PNPM
-        uses: pnpm/action-setup@v2.0.1
+        uses: pnpm/action-setup@v2.2.1
         with:
-          version: latest
-          run_install: false
+          version: 7.0.0
```
Comment (LitoMore) [inline .github/workflows/deploy-storybook.yml]: Ditto for all files.

Comment (BlackHole1) [reply .github/workflows/deploy-storybook.yml]: Okey

**Review by hyrious — DISMISSED** [review]

**Review by crimx — DISMISSED** [review]

**Comment by BlackHole1** [comment]
Waiting https://github.com/netless-io/flat/pull/1512 Merged

**Review by Cheerego7 — DISMISSED** [review]

**Pull request merged**